aiosmtpd/tests/test_smtp.py

```diff
@@ -1154,9 +1344,8 @@ def test_help_authenticated(self):
                 b64encode(b'\0goodlogin\0goodpasswd').decode()
             )
             assert_auth_success(self, code, response)
-            code, response = client.docmd('HELP')
-            self.assertEqual(code, 250)
-            self.assertEqual(response, SUPPORTED_COMMANDS_NOTLS)
```

Comment:
Again, change from piece-by-piece comparison to direct tuple-vs-tuple comparison, in order to not hide possibly helpful error messages.

Also, migrate to pytest's `assert` syntax.